readme.md

```diff
@@ -56,26 +56,28 @@ Note: You're not restricted to digital ocean. Any machine with docker should be
 
        $ git clone https://github.com/speckleworks/SpeckleServer.git
        $ cd SpeckleServer
-       $ npm install
+       $ sudo npm install
 
-5) Follow the instructions in `.env-base` file to configure your server.
+5) Follow the instructions in `.env-base` file to configure your server. Use `sudo nano .env` to edit.
```

Comment:
again, theoretically, if you're logged in as a non-sudo user, you should be able to edit the files from your home dir, where theoretically you've cloned the repo...

Reply:
as a non-admin user I got 'access restricted' when trying to save canges - was only possible with adding sudo to the command